api/server/router/system/backend.go

```diff
@@ -12,7 +12,7 @@ import (
 type Backend interface {
 	SystemInfo() (*types.Info, error)
 	SystemVersion() types.Version
-	SubscribeToEvents(since, sinceNano int64, ef filters.Args) ([]events.Message, chan interface{})
+	SubscribeToEvents(since, sinceNano, until, untilNano int64, ef filters.Args) ([]events.Message, chan interface{})
```

Comment:
WDYT about passing `time.Time` values here for `since` and `until`? It feels like it would be a slightly cleaner interface. And `getEvents` is already calling `time.Unix` on `until`...